lib/tasks/migrate_data.rake

```diff
@@ -0,0 +1,18 @@
+def create_commits!(repository)
+  repository.git.commits { |c| repository.commit_for!(c.oid) }
+end
+
+def attach_commit!(ontology_version)
+  commit = ontology_version.repository.commits.
+    find_by_commit_oid(ontology_version.commit_oid)
```

Comment:
We end on line-continuation with `.` as it is in our style guide. It is in our style guide because `.` at the end of the line instead of the beginning allows for comments between those two lines. When placing the `.` at the beginning of the line comments would result in syntax errors.